scripts/inclusion-bot.yaml

```diff
@@ -292,7 +294,7 @@ triggers:
     alternatives:
       - antipattern
       - anti-UX pattern
-      - consequence design
+      - deceptive design pattern
```

Comment:
suggest keeping "consequence design". Thanks to @quarterback it's rapidly becoming the preferred terminology. I like deceptive design pattern, too. Why not both?

Reply:
In the Inclusion Bot doc, consequence design was listed with a link, which indicated that it wasn't commonly used (i.e. we'd be suggesting a term that folks would need to do further research to understand). If it is commonly understood, definitely keep.

Reply:
My understanding is that it's not ***yet*** a well-known term. Having talked to Ron about it, I suspect it'll be common terminology before too long (because it makes so darn much sense). I'd probably favor striking it for now and creating an issue to add it back in a few months. Though I might also look to @quarterback to see if he has a better sense of how widely adopted the term is, since AFAIK he coined it!